src/Features/CSharp/Portable/CodeFixes/Async/CSharpAddAwaitCodeFixProvider.cs

```diff
@@ -98,7 +98,7 @@ private static bool DoesExpressionReturnTask(ExpressionSyntax expression, Semant
             }
 
             return TryGetExpressionType(expression, semanticModel, out var returnType) &&
-            semanticModel.Compilation.ClassifyConversion(taskType, returnType).Exists;
+            semanticModel.Compilation.ClassifyConversionInfo(taskType, returnType).Exists;
```

Comment:
From this point on, i just went to lots of the IDE that looks at conversions and i ported it over to use the new API.  When i discovered cases of identical VB/C# code, i merged the code into the base-type.  I expect there will be a lot more opportunities for shared IDE code with this entire refactoring.